client/me/purchases/remove-plan-dialog/get-plan-features.tsx

```diff
@@ -0,0 +1,257 @@
+	const shippingCarriers = String( translate( 'Integration with top shipping carriers' ) );
+	const premiumDesign = String(
+		translate( 'Premium design options customized for online stores' )
+	);
+	const more = String( translate( 'and more' ) );
+
+	/**
+	 * Personal plan
+	 */
+	if ( isWpComPersonalPlan( productSlug ) ) {
+		// Yearly plan
+		if ( ! isMonthlyPlan ) {
+			// Without domain
+			if ( ! hasDomain ) {
+				return [
+					freeDomainFirstYear,
+					bestInClassHosting,
+					addFreeSite,
+					collectPayments,
+					emailCustomerSupport,
+				];
+			}
+
+			// With domain within first year
+			if ( withinFirstYear ) {
+				return [
+					freeDomain,
+					bestInClassHosting,
+					addFreeSite,
+					collectPayments,
+					emailCustomerSupport,
+				];
+			}
+
+			// With domain post first year
+			return [ bestInClassHosting, addFreeSite, collectPayments, emailCustomerSupport ];
+		}
+
+		// Monthly plan
+		return [ bestInClassHosting, addFreeSite, collectPayments, emailCustomerSupport ];
+	}
+
+	/**
+	 * Premium plan
+	 */
+	if ( isWpComPremiumPlan( productSlug ) ) {
+		// Yearly plan
+		if ( ! isMonthlyPlan ) {
+			// Without domain
+			if ( ! hasDomain ) {
+				return [
+					freeDomainFirstYear,
+					liveChat,
+					earnAndRevenue,
+					premiumThemes,
+					analyticsIntegration,
+					bestInClassHosting,
+					addFreeSite,
+				];
+			}
+
+			// With domain within first year
+			if ( withinFirstYear ) {
+				return [
+					freeDomain,
+					liveChat,
+					earnAndRevenue,
+					premiumThemes,
+					analyticsIntegration,
+					bestInClassHosting,
+					addFreeSite,
+					collectPayments,
+				];
+			}
+
+			// With domain post first year
+			return [
+				liveChat,
+				earnAndRevenue,
+				premiumThemes,
+				analyticsIntegration,
+				bestInClassHosting,
+				addFreeSite,
+				collectPayments,
+			];
+		}
+
+		// Monthly plan
+		return [
+			earnAndRevenue,
+			premiumThemes,
+			analyticsIntegration,
+			bestInClassHosting,
+			addFreeSite,
+			collectPayments,
+			emailCustomerSupport,
+		];
+	}
+
+	/**
+	 * Business plan
+	 */
+	if ( isWpComBusinessPlan( productSlug ) ) {
+		// Yearly plan
+		if ( ! isMonthlyPlan ) {
+			// Without domain
+			if ( ! hasDomain ) {
+				return [
+					freeDomainFirstYear,
+					accessPlugins,
+					seoTools,
+					automatedBackups,
+					sftpAndDatabase,
+					liveChat,
+					more,
+				];
+			}
+
+			// With domain within first year
+			if ( withinFirstYear ) {
+				return [
+					freeDomain,
+					accessPlugins,
+					seoTools,
+					automatedBackups,
+					sftpAndDatabase,
+					liveChat,
+					more,
+				];
+			}
+
+			// With domain post first year
+			return [
+				accessPlugins,
+				seoTools,
+				automatedBackups,
+				sftpAndDatabase,
+				bestInClassHosting,
+				liveChat,
+				more,
+			];
+		}
+
+		// Monthly plan
+		return [
+			accessPlugins,
+			seoTools,
+			automatedBackups,
+			sftpAndDatabase,
+			bestInClassHosting,
+			liveChat,
```

Comment:
Does `liveChat` belong here?  It's not listed for the other monthly plans, and seems to contradict https://wordpress.com/pricing/.

Reply:
Features were updated in the design vTj2aknM6tRUQndWtOcURT-fi-445%3A10488